examples/z_sub_thr.c

```diff
@@ -21,43 +21,48 @@
 typedef struct {
     volatile unsigned long count;
     volatile unsigned long finished_rounds;
-    volatile clock_t start;
-    volatile clock_t stop;
-    volatile clock_t first_start;
+    struct timespec start;
+    struct timespec first_start;
 } z_stats_t;
 
 z_stats_t *z_stats_make() {
     z_stats_t *stats = malloc(sizeof(z_stats_t));
     stats->count = 0;
     stats->finished_rounds = 0;
-    stats->first_start = 0;
+    stats->first_start.tv_nsec = 0;
     return stats;
 }
 
+static inline unsigned long elapsed_us(const struct timespec *start, const struct timespec *end) {
```

Comment:
It may be better to immediately return floating point seconds than to convert them further each time at the risk of making a mistake with the number of zeros.